lib/src/generateKernels.cc

```diff
@@ -261,12 +257,18 @@ void genNeuronKernel(const NNmodel &model, //!< Model description
             name_substitutions(psCode, "", nModels[nt].extraGlobalNeuronKernelParameters, model.neuronName[i]);
             psCode= ensureFtype(psCode, model.ftype);
             checkUnreplacedVariables(psCode,"postSyntoCurrent");
-            os << psCode << ";" << ENDL;
+            if (!psm.supportCode.empty()) {
+                os << OB(29) << " using namespace " << sName << "_postsyn;" << ENDL;
+            }
+            os << "Isyn += " << psCode << ";" << ENDL;
             if (!psm.supportCode.empty()) {
                 os << CB(29) << " // namespace bracket closed" << ENDL;
             }
         }
 
+	if (!nModels[nt].supportCode.empty()) {
+	    os << " using namespace " << model.neuronName[i] << "_neuron;" << ENDL;
+	}
```

Comment:
Am I missing something or is this namespace not getting closed (I guess it should be closed around line 365)?